### ISSUE TITLE
Bug Report: Astra Militarum, Cadian Detachment, Knight Commander Pask.

### ISSUE THREAD

**Issue by SilverStrike16** [BSData/wh40k-7th-edition#1671]
When making a Cadian detachment as per Mont'ka rules, I can make an Emperor's Wrath Armoured Company, but I can't then upgrade the tank commander of that group to be Pask. It's not user error, I made a normal AM army and can add Pask, but the same input method doesn't work for the Cadian variant.

Hopefully an easy fix, Pask is an auto-field for my armies!

**Comment by ecrousseau** [comment]
I don't believe Knight Commander Pask is an option within the Armoured Company - special characters are generally not allowed within formations (and he is specifically listed as an option for the Battle Group Command, implying that he would be mentioned within the Armoured Company if he were an available option there).

I would be happy to hear opposing viewpoints/evidence on the issue though :)

**Comment by SilverStrike16** [comment]
He's an upgrade choice for any tank commander. A character upgrade, but still an upgrade, similar to taking sponsons or extra armour. There are no restrictions on the formation, it's called a Tank Commander for the formation... And, most importantly, the actual picture itself of the Emperor's Fist Armoured Company has Pask sat right there in the foremost Leman Russ :P So I'm reasonably certain he's allowed. Checked my own copy of Mont'ka. Plus, in the lore book of the big battle and all that business against the Tau, he is actually in the battle with the Cadians doing Pask things and surviving.

**Comment by ecrousseau** [comment]
The stumbling block for me is that he is specifically listed (see footnote 2) for the Battle Group Command. I don't see why they'd do that if you could take him as an upgrade for the other Tank Commander entries?

In any case, since the default position is to have the data files be as flexible as possible, I'll make a change to allow Pask in the Armoured Company.

**Comment by epvanhouten** [comment]
I agree with ecr that the rules appear to read as he is allowed in the BGC
but not in the Armored Company. I think it is a wrong to alter it.

Note that you are not taking an Armored Company for the BGC, just the tank
commander. Using the logic of allowing pask as a substitute in the Armored
Copany would also allow Creed and Kell to be taken in any of the formations
that have a Company Command Squad in them.

**Comment by SilverStrike16** [comment]
I'd be quite happy if it were altered to allow it, especially given he's right there in the picture of the Armoured Company. Sure it says he can be taken for the BGC, but there's nothing to restrict him from being taken in the AC. Especially given he doesn't necessarily have to be a detachment's warlord, so he can be in positions other than the direct leader.

The Tank Commander data sheet allows it to upgrade to Pask, so as written I would say he does not have to be in the Command Choice tank commander and can be taken in the Armored company.

**Comment by epvanhouten** [comment]
The last line of the Restrictions rules for the detachment is 'No unique
models other then those listed here may be taken as part of this
Detachment.' That seems pretty cut and dry.

**Comment by amis92** [comment]
I think @epvanhouten hit it. As such, he should not be allowed, as it currently is. Not sure, however, what it'd be ruled if the Armored Company was taken as a Formation by itself.

**Comment by SilverStrike16** [comment]
The same discussion came up on the dakkadakka forums relating to the space marines.
when the New Space Marine Codex came out with the Gladius Strike force(sort of).
In the Gladius, the Command section lists the various Chapter Master Special Characters as options, The Demi-Company Formation only lists the Captain Special characters. Both were options in place of the Captain. The Poster had asked if that meant that you could not upgrade a Captain to a Chapter Master in the Demi-company formation (yes, you can; you are still taking a Captain Unit Entry, and a Captain Unit, you are just using a different model).

So by the same logic, Pask can be taken in an armoured company group.

I mean at the end of the day, I'm still going to be doing it. What I was really hoping was that this shiny new app that I had come across, and found really useful, would be able to reflect what I'm taking. If it's a distinct case of 'some think so, some think not' (As another thread on belloflostsouls also share my point of view and would allow Pask), maybe it would be kinder to add it in for those who will use it? And those who don't, well, no big deal. At the end of the day it's an app that's meant to be a useful utility to army builders. No reason not to have it, especially since it's such a minor adjustment.

Basically.. Please could it be adjusted to be doable? :)

**Comment by amis92** [comment]
You've ignored the restriction referenced. I think it's not the same case. But it's not up to me. Just making a point here.

**Comment by ecrousseau** [comment]
I've made the change. Until there is an FAQ or similar on the subject, I think its best to leave the option open.
